pkg/workflows/sdk/workflow_spec.go

```diff
@@ -35,3 +153,61 @@ func (w *WorkflowSpec) Steps() []StepDefinition {
 	s = append(s, w.Targets...)
 	return s
 }
+
+func (w *WorkflowSpec) FormatChart() (string, error) {
```

Comment:
Hey @jmank88 , I am trying to figure out how we provide the visual to users via CLI. From the dev platform CLI perspective, we compile users' workflow and build a wasm for them. We have the wasm binary and config as byte[], doesnt it make sense to have an interface to take the byte[] of wasm and config, and generate the diagram? It should be similar to this non-visual interface: https://github.com/smartcontractkit/chainlink-common/blob/b7b7f6310ac2815acf7a4deae0d3c5e33ac349d9/pkg/workflows/wasm/host/wasm.go#L15

Reply:
Let's move this discussion over to #790, which just introduces the chart without any of this builder stuff.